src/components/GameBoard.js

```diff
@@ -0,0 +1,123 @@
+import React, { useState, useEffect } from 'react';
+import { Card } from './Card'
+import '../styles/gameBoard.scss'
+import { v4 as uuidv4 } from 'uuid';
```

Comment:
Nit picking. Import order normally would be
1. external library installed through npm
2. Internal non component files
3. Internal components
4. css, images

In this case:
```suggestion
import React, { useState, useEffect } from 'react';
import { v4 as uuidv4 } from 'uuid';

import { Card } from './Card'
import '../styles/gameBoard.scss'
```
You don't have to follow exactly but at least separate external and internal imports, and keep css file at the end.